adc/consumer.py

```diff
@@ -86,6 +97,26 @@ def mark_done(self, msg: confluent_kafka.Message, asynchronous: bool = True):
         else:
             self._consumer.commit(msg, asynchronous=False)
 
+    def _offsets_for_position(self, partitions: Collection[confluent_kafka.TopicPartition],
```

Comment:
This fails on python versions <3.9 with
```TypeError: 'ABCMeta' object is not subscriptable```
The fix seems to be to import `Collection` from `typing` (which is more lax about what it allows) instead of `collections.abc`.